packages/client/components/TeamPrompt/TeamPromptDiscussionDrawer.tsx

```diff
@@ -34,7 +34,6 @@ const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop,
   top: 0,
   right: isDesktop ? 0 : undefined,
   transition: `all 200ms ${BezierCurve.DECELERATE}`,
-  userSelect: 'none',
```

Comment:
@jmtaber129 do you remember why it was added here in the first place? I checked and I haven't found any regression so it should be good. another fix would be to add `userSelect: 'auto'` to the response editor itself, but this feels more appropriate, unless I'm missing why it's here

Reply:
I think this was largely copied from another drawer in the app, I think for the case when the drawer is draggable (smaller screens where the drawer appears on top of the main view IIRC). So without this, a user trying to drag the drawer closed might end up selecting text.

We can probably just condition this style on `isDesktop` being false

Reply:
done, @jmtaber129 could you take a look now?